### ISSUE TITLE
[dashing] deb is empty due to missing patches

### ISSUE THREAD

**Issue by mikaelarguedas** [ros2-gbp/ros1_bridge-release#4]
The package doesnt build on dashing due to missing dependencies, from http://build.ros2.org/view/Dbin_uB64/job/Dbin_uB64__ros1_bridge__ubuntu_bionic_amd64__binary/15/consoleFull:

```
-- Checking for module 'roscpp'
--   No package 'roscpp' found
CMake Warning at CMakeLists.txt:34 (message):
  Failed to find ROS 1 roscpp, skipping...
```

It looks like the [patches](https://github.com/ros2-gbp/ros1_bridge-release/tree/patches/debian/crystal/ros1_bridge) to inject dependencies are missing

**Comment by nuclearsandwich** [comment]
Fixed in a680081f8004f19eeec372665e62fefcf70e3845 and released with https://github.com/ros/rosdistro/pull/21465. This will be in the next Dashing sync.

**Comment by mikaelarguedas** [comment]
Great thanks :bowing_man:
